x-pack/plugins/cases/common/types/api/case/v1.ts

```diff
@@ -214,7 +221,15 @@ export const CasesFindRequestRt = rt.intersection([
       /**
        * The category of the case.
        */
-      category: rt.union([rt.array(rt.string), rt.string]),
+      category: rt.union([
+        limitedArraySchema({
+          codec: rt.string,
```

Comment:
Should this be limited to 50 characters too? (like in the post request)

Reply:
As this is a find request and is not persisted in ES I think is fine to not limit it (same as we do with the tags).